website/public/benchmarks/index.ejs.html

```diff
@@ -95,7 +95,9 @@ <h2>Methodology</h2>
         <p>These benchmarks measure the following linters:</p>
         <ul>
           <li>
-            <strong class="linter-name" style="--hue: 0">quick-lint-js</strong>
+            <strong class="linter-name" style="--hue: 0"
```

Comment:
Why these whitespaces changes? Are they significant? Or am I missing something?

Reply:
hmm not sure, ive been running prettier. Ill fix that.